pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -759,12 +759,6 @@ func (r *RepoSyncReconciler) mapConfigMapToRepoSyncs(obj client.Object) []reconc
 func (r *RepoSyncReconciler) mapObjectToRepoSync(obj client.Object) []reconcile.Request {
 	objRef := client.ObjectKeyFromObject(obj)
 
-	// Ignore changes from other namespaces because all the generated resources
-	// exist in the config-management-system namespace.
-	if objRef.Namespace != configsync.ControllerNamespace {
```

Comment:
Should we perhaps keep this early exit for non-RoleBindings (Deployment and ServiceAccount)?

Reply:
done